character-controller.js

```diff
@@ -159,8 +159,14 @@ class PlayerBase extends THREE.Object3D {
       app.parent && app.parent.remove(app);
     });
 
+    this.eyeTarget = new THREE.Vector3();

```

Comment:
Is this `eyeTarget` or `headTarget`. Does it need a rename?

Reply:
Yes, it's `headTarget` indeed, `eyeTarget` is just inherited from avatar's old name.
Let me rename it.

Reply:
@avaer Renamed https://github.com/webaverse/app/pull/2949/commits/32dcb342180ac144e8865331f76555334e39b903